xonsh/imphooks.py

```diff
@@ -295,13 +313,10 @@ def exec_module(self, module):
         events.on_import_post_exec_module.fire(module=module)
         return rtn
 
-    def load_module(self, fullname):
-        """Legacy module loading, provided for backwards compatibility."""
-        return self.loader.load_module(fullname)
-
-    def module_repr(self, module):
-        """Legacy module repr, provided for backwards compatibility."""
-        return self.loader.module_repr(module)
+    def __getattribute__(self, name):
```

Comment:
Why this over `__getattr__`?

Reply:
Oops! That's my fault. @astronouth7303 is right - `__getattr__` is the one we need.

Reply:
Argh, right! It worked and I did not think twice...